### PR TITLE
[TT-10662] Update READMEs 

### PR DESCRIPTION
**Pull request by buraksekili**
## Description

This PR updates chart READMEs

## Related Issue

https://tyktech.atlassian.net/browse/TT-10662

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] Documentation updates or improvements.

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [ ] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
     - [ ] I have manually updated the README(s)/documentation accordingly.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: components/tyk-dashboard/README.md [hunk 1]
```diff
@@ -16,70 +16,89 @@ For typical usage, we recommend using following umbrella charts:
...
-    helm show values tyk-helm/tyk-dashboard > values-dashboard.yaml --devel
+```bash
+helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+helm show values tyk-helm/tyk-dashboard > values-dashboard.yaml
+```
 
 Note: 
-* Set redis connection details first at `.Values.global.redis`
-* Set mongo connection details second at `.Values.global.mongo`
+* Set Redis connection details first at `.Values.global.redis`
```
Comment (komalsukhani) [inline components/tyk-dashboard/README.md:31]:
```suggestion
* Set Redis connection details at `.Values.global.redis`
```

File: components/tyk-dashboard/README.md [hunk 2]
```diff
@@ -16,70 +16,89 @@ For typical usage, we recommend using following umbrella charts:
...
+* Set MongoDB connection details second at `.Values.global.mongo`
```
Comment (komalsukhani) [inline components/tyk-dashboard/README.md:32]:
```suggestion
* Set MongoDB connection details at `.Values.global.mongo`
```

File: tyk-oss/README.md [hunk 3]
```diff
@@ -230,19 +230,21 @@ If you want to use your own key/cert pair, you must follow the following steps:
 An Ingress resource is created if `tyk-gateway.gateway.ingress.enabled` is set to true.
 
 ```yaml
+    # Creates an ingress object in k8s. Will require an ingress-controller and
+    # annotation to that ingress controller.
     ingress:
       # if enabled, creates an ingress resource for the gateway
-      enabled: true
+      enabled: false
```
Comment (komalsukhani) [inline tyk-oss/README.md:237]: Shouldn't it be set to true since we are providing an example?

**Review by komalsukhani — CHANGES_REQUESTED** [review]

File: tyk-oss/README.md (hunk 3 above)
Comment (buraksekili) [reply tyk-oss/README.md:237]: yes, it must be true

Comment (buraksekili) [reply tyk-oss/README.md]: updated accordingly in https://github.com/TykTechnologies/tyk-charts/pull/191/commits/c2e0cfc0ad304dfd75751be442378f9eb16f3af9

File: tyk-stack/README.md [hunk 4]
```diff
@@ -448,105 +466,214 @@ Uptime Pump can be configured by setting `pump.uptimePumpBackend` in values-stac
...
-<!-- END import from pump doc -->
-
-
 #### Tyk Dashboard
-The Tyk Dashboard can be configured by modifying the values under "tyk-dashboard" section of the `values-stack.yaml` file
+The Tyk Dashboard can be configured by modifying the values under `tyk-dashboard` section of the `values-stack.yaml` file
 The chart is provided with sane defaults such that the only hard requirement is the license which needs to be put under
 `.Values.global.license.dashboard` in order for the bootstrapping process to work.
 
 ```yaml
```
[the remark this answers is not in the document]
Comment (komalsukhani) [reply tyk-stack/README.md:474]: Yeah. That sounds good.

**Review by komalsukhani — APPROVED** [review]

**Pull request merged**